src/peer/negotiator.js

```diff
@@ -595,9 +612,6 @@ class Negotiator extends EventEmitter {
   _replacePerStream(newStream) {
     const localStreams = this._pc.getLocalStreams();
 
-    // Temporarily unset onnegotiationneeded so that it doesn't do anything.
-    // Leaving this set will cause an extra negotiation on removeStream that will cause the
-    // signalingState on the answer side to enter an unexpected state, leading to errors.
     const origOnNegotiationNeeded = this._pc.onnegotiationneeded;
     this._pc.onnegotiationneeded = () => {};
```

Comment:
We could probably refactor this logic using the new `_isNegotiationAllowed` flag but the obvious fix didn't work.
Since this code will be deleted once we deprecate Chrome 64, I don't think there's too much point in simplifying.